mobile/src/main/java/org/openhab/habdroid/model/CloudNotification.kt

```diff
@@ -35,10 +36,10 @@ data class CloudNotification internal constructor(
 fun JSONObject.toCloudNotification(): CloudNotification {
     var created: Long = 0
     if (has("created")) {
-        val format = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'")
+        val format = SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.S'Z'", Locale.US)
         format.timeZone = TimeZone.getTimeZone("UTC")
         try {
-            created = format.parse(getString("created")).time
+            created = format.parse(getString("created"))?.time  ?: 0
```

Comment:
Excess space character.

Besides that, maybe instead of the has() call above, do getString() earlier and skip if result is null?

Reply:
`parse()` may return null, so I need this elvis even if I move `JSONObject.getString()` above.